### PR TITLE
#892 NOTES - INTERVIEW, Add emergency questions

### PR DESCRIPTION
**Pull request by MDR22**
Added emergency questions to script:
- Added separate dialog tab for emergency questions with error handling
- Updated save your work functionality for emergency questions
- Updated script lowdown to include emergency questions
- Responses to emergency questions will write to PDF and CASE/NOTE

### CODE REVIEW

**Review by MeganGeissler — APPROVED** [review]
Works well! - since the EMER error handling is only applicable if the EMER box being checked at the beginning of the script, wondering if we only want the EMER Q tab to only show up when the EMER box is checked, or do we always want it available?

**Comment by MDR22** [comment]
> Works well! - since the EMER error handling is only applicable if the EMER box being checked at the beginning of the script, wondering if we only want the EMER Q tab to only show up when the EMER box is checked, or do we always want it available?

My understanding is that we want the questions to be required if the EMER box is checked, but otherwise they are optional so that they still appear if the resident reports an emergency during the interview.

**Review by courtrightd — COMMENTED** [review]
Combobox for emergency_type should have "Select or type..."
When typing in my own answer to that field, it loses it when going to different dialog, and then get the must complete questions error. Perhaps remove that from required fields, so that if the "emer" doesn't fit within program policy we can explain it in the discussion box.

**Comment by MDR22** [comment]
> Combobox for emergency_type should have "Select or type..." When typing in my own answer to that field, it loses it when going to different dialog, and then get the must complete questions error. Perhaps remove that from required fields, so that if the "emer" doesn't fit within program policy we can explain it in the discussion box.

Switched combobox to have 'Select or type' option. Resolved issue of script resetting custom combobox entry when navigating away from EMER Q dialog.

**Review by courtrightd — APPROVED** [review]
Approved!

**Review by C-Love — CHANGES_REQUESTED** [review]
We should try to simplify the options for the interview question.

File: notes/interview.vbs [hunk 1]
```diff
@@ -1788,8 +1798,28 @@ function define_main_dialog()
...
+				btn_pos = btn_pos + 15
+			End If
+			
+			Text 505, btn_pos+2, 60, 10, "EMER Q"
+
+			GroupBox 5, 5, 475, 115, "Emergency Details"
+			Text 15, 20, 145, 10, "Is the resident experiencing an emergency?"
+			DropListBox 190, 20, 30, 15, " "+chr(9)+"Yes"+chr(9)+"No", resident_emergency_yn
+			Text 15, 40, 160, 10, "What emergency is the resident is experiencing?"
+			ComboBox 190, 40, 90, 25, "Select or type"+chr(9)+"Bus Tickets"+chr(9)+"Damage Deposit"+chr(9)+"Eviction Notice"+chr(9)+"First Month's Rent"+chr(9)+"Foreclosure"+chr(9)+"Home Repairs"+chr(9)+"Moving Expenses"+chr(9)+"Previous Rent"+chr(9)+"Property Taxes"+chr(9)+"Utility Bills/Disconnect"+chr(9)+emergency_type, emergency_type
```
[the remark this answers is not in the document]
Comment (MDR22) [reply notes/interview.vbs]: Updated the dialog fields accordingly - thanks!

**Review by courtrightd — APPROVED** [review]
Thanks for the updates!

**Pull request merged**